manager/allocator/network.go

```diff
@@ -768,13 +897,29 @@ func (a *Allocator) commitAllocatedNode(ctx context.Context, batch *store.Batch,
 
 		if err == store.ErrSequenceConflict {
 			storeNode := store.GetNode(tx, node.ID)
-			storeNode.Attachment = node.Attachment.Copy()
+
+			storeNodeLBMap := make(map[string]*api.NetworkAttachment)
+			for _, na := range storeNode.LbAttachments {
+				storeNodeLBMap[na.Network.ID] = na
+			}
+
+			for _, na := range node.LbAttachments {
+				if _, ok := storeNodeLBMap[na.Network.ID]; !ok {
+					//PR-Question: Do we need to NetworkAttachment.Copy() the attachments?
```

Comment:
@nishanttotla : Do we need to copy each network attachment?  Or can I just do:
storeNode.LBAttachemtns = node.LbAttachments.
It is not clear why I need to make a copy since a copy will be done when the node is updated in the store.

Reply:
It's not necessary to copy them here because `UpdateNode` will automatically make a deep copy of the node.